screens/modals/ResponderHelpRequestModal.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from "react";
+import {
+  StyleSheet,
+  Button
+} from "react-native";
+import { Actions } from "react-native-router-flux";
+import { Container, Content, Header, View } from "../../components/layout";
+import { Text } from "../../components/typography";
+
+const ResponderHelpRequestModal = (props) => {
+
```

Comment:
```suggestion
  if(AppState.currentState === 'background') return null
```

Might not be ideal solution tho. (_AppState_ is imported from react-native)

Reply:
I will make this change for now and make a TODO